### DISCUSSION TITLE
NB-Group的更新频道

### DISCUSSION THREAD

**Discussion by NB-Group** [win12-online/win12#97] [Show and tell]
我在这里发更新

**Comment by NB-Group** [comment]
准备做下windows安全中心。
大概样子：
![image](https://github.com/tjy-gitnub/win12/assets/121747915/f6e6ffbe-a0ef-4060-b8d4-08eb82c815f8)
过会优化下css

**Reply by User782Tec** [reply]
不错

建议照着设置/文件资源管理器来做

**Reply by NB-Group** [reply]
> 不错
> 
> 建议照着设置/文件资源管理器来做

好滴

**Reply by User782Tec** [reply]
还有
这玩意为啥不是中文滴

**Reply by NB-Group** [reply]
> 还有 这玩意为啥不是中文滴

装到win12的时候肯定是中文的啦

**Comment by NB-Group** [comment]
<img width="203" alt="image" src="https://github.com/tjy-gitnub/win12/assets/121747915/0456addf-7cb4-459f-9cd7-faaeda5e2530">

这个小竖线怎么搞的
@User782Tec

**Comment by NB-Group** [comment]
![image](https://github.com/tjy-gitnub/win12/assets/121747915/c01b790b-6375-423f-82a0-fb4aeef21974)
fuck（只要窗口不是活动窗口就会这样，活动窗口到是正常）

**Comment by NB-Group** [comment]
<img width="955" alt="image" src="https://github.com/tjy-gitnub/win12/assets/121747915/ae8a061d-42be-44c8-b9cb-956cf4d49632">
<img width="946" alt="image" src="https://github.com/tjy-gitnub/win12/assets/121747915/70615e1e-033d-40af-981e-54c1309dea13">
mmp的，fuck，就一个模糊的css居然让整个侧边栏不见了

**Reply by NB-Group** [reply]
貌似是因为我用fixed定位的原因欸

**Reply by NB-Group** [reply]
哦吆西，貌似是因为我用top定位的问题去掉之后就行了

**Comment by NB-Group** [comment]
![image](https://github.com/tjy-gitnub/win12/assets/121747915/dbbae485-f448-4730-8220-c586a6000971)
这个小竖线怎么搞的
@User782Tec

**Reply by tjy-gitnub** [reply]
就是一个比较高的div啊

**Reply by NB-Group** [reply]
> 就是一个比较高的div啊

我的意思是怎么做到让他和已选择的侧边栏按钮一起动啊

**Reply by tjy-gitnub** [reply]
欧，就是用 `position: relative;` 啊
可以直接在desktop里面用
```html
<list class="focs">
<a class="a">...</a>
<a class="a">...</a>
<span class="focs"></span>
</list>
```
不需要自己写

**Reply by NB-Group** [reply]
> Comment options
> ####
> [ tjy-gitnub ](/tjy-gitnub) [ May 28, 2023 ](#discussioncomment-6021854)
> Maintainer

哦~吆西

**Reply by tjy-gitnub** [reply]
选中的需要加一个 `.check`

**Reply by NB-Group** [reply]
> 选中的需要加一个 `.check`

了然了然

**Reply by NB-Group** [reply]
欸，不对啊，为啥加上了也莫得小竖线啊。
<img width="725" alt="image" src="https://github.com/tjy-gitnub/win12/assets/121747915/7b633be1-810d-42b9-bfe5-97af1cf56edf">
@tjy-gitnub

**Reply by tjy-gitnub** [reply]
啊 理论上应该有啊，应该是显示问题，你调一下span的显示和位置呢

**Reply by NB-Group** [reply]
> 啊 理论上应该有啊，应该是显示问题，你调一下span的显示和位置呢

我康康

**Reply by User782Tec** [reply]
这玩意有点缺陷（或者说兼容性不好）
需要写更多的代码才能正常使用
（我今天写任务管理器的时候就弄了好久，照着设置终于给写好了）

**Reply by tjy-gitnub** [reply]
额，又不是给用户用

**Comment by NB-Group** [comment]
现在还可以吧？
![image](https://github.com/tjy-gitnub/win12/assets/121747915/ee66b0c9-0fc4-4179-a5ea-8ae5c9d252c9)

**Reply by User782Tec** [reply]
> 现在还可以吧？ ![image](https://user-images.githubusercontent.com/121747915/241749614-ee66b0c9-0fc4-4179-a5ea-8ae5c9d252c9.png)

为啥不三个一排，分两排呢

**Reply by NB-Group** [reply]
> > 现在还可以吧？ ![image](https://user-images.githubusercontent.com/121747915/241749614-ee66b0c9-0fc4-4179-a5ea-8ae5c9d252c9.png)
> 
> 为啥不三个一排，分两排呢

问得好！

**Reply by tjy-gitnub** [reply]
额，只能说将就吧

**Comment by NB-Group** [comment]
![image](https://github.com/tjy-gitnub/win12/assets/121747915/48fa6483-10ae-46b8-b0ee-77d65cd1c4b5)
酱几，对吧

**Reply by User782Tec** [reply]
> ![image](https://user-images.githubusercontent.com/121747915/241942925-48fa6483-10ae-46b8-b0ee-77d65cd1c4b5.png) 酱几，对吧

对的

**Reply by tjy-gitnub** [reply]
我个人认为win11的会好看点
还有颜色
![image](https://github.com/tjy-gitnub/win12/assets/71509955/cc5b0f15-604d-4a55-a443-137432ecf86a)
#2983cc 或者 var(--href)

**Reply by NB-Group** [reply]
> 我个人认为win11的会好看点 还有颜色#2983cc 或者 var(--href)

咳咳咳，这个是从源文件里扒出来的图片，直接就是黑白的，win11理论上也可以在C:\Windows\SystemApps\Microsoft.Windows.SecHealthUI_cw5n1h2txyewy\Assets
找的到

![image](https://github.com/tjy-gitnub/win12/assets/121747915/b09c9ff0-329a-43fc-a323-7194395bc2bd)

**Reply by tjy-gitnub** [reply]
png!!!! 6
好吧，我以为是svg...

**Reply by tjy-gitnub** [reply]
&#xEA18;
&#xE77B;
&#xEC44;
&#xE737;
&#xE770;
&#xE95E;
不谢

**Reply by tjy-gitnub** [reply]
`&#xEA18;` `&#xE77B;` `&#xEC44;` `&#xE737;` `&#xE770;` `&#xE95E;`

**Reply by NB-Group** [reply]
>       不谢

哇，你咋找的

**Comment by NB-Group** [comment]
![image](https://github.com/tjy-gitnub/win12/assets/121747915/52218c8f-ae18-44f8-9a72-9d944749f60a)
我做的这个进度条好丑。。。你们有啥能给我借鉴的代码吗

**Reply by User782Tec** [reply]
![image](https://github.com/tjy-gitnub/win12/assets/115236204/b25fb910-3447-4c53-b28f-451aeee4c185)

呐

**Reply by NB-Group** [reply]
> ![image](https://user-images.githubusercontent.com/115236204/241977975-b25fb910-3447-4c53-b28f-451aeee4c185.png)
> 
> 呐

代。。代码？

**Reply by User782Tec** [reply]
> > ![image](https://user-images.githubusercontent.com/115236204/241977975-b25fb910-3447-4c53-b28f-451aeee4c185.png)
> > 呐
> 
> 代。。代码？

等我发v5.3.0(doge

CSS: 
```css
.range-container {
    width: 100%;
    height: 4.5px;
    background-image: linear-gradient(100deg, var(--theme-1), var(--theme-2));
    border-radius: 7px;
}

.range-container>.after {
    border-radius: 0px 7px 7px 0px !important;
}

.range-container>.after {
    height: 4.5px;
    position: relative;
    background-color: var(--bg70);
    bottom: 0px;
    width: 0px;
    border-radius: 7px;
}
```

HTML: 
```html
<div class="range-container">
    <div class="after"></div>
</div>
```

然后随便用js写写进度调整就能用了
`.after`元素是后面的遮罩

**Reply by NB-Group** [reply]
搞了半天显示不出来才发现--theme-1

**Reply by tjy-gitnub** [reply]
用js?太奢侈了吧，要是我直接css的animation...

**Reply by NB-Group** [reply]
> 用js?太奢侈了吧，要是我直接css的animation...

css动画也可以的的哦！我傻了

**Reply by User782Tec** [reply]
> 用js?太奢侈了吧，要是我直接css的animation...

貌似确实
我是因为要获取电量调整的
所以用的js

**Comment by NB-Group** [comment]
![image](https://github.com/tjy-gitnub/win12/assets/121747915/5c5999c6-b825-4392-8938-9d23bb9095ae)
![image](https://github.com/tjy-gitnub/win12/assets/121747915/05509260-5234-4b44-b8d1-b1981db5c21e)

本次更新的重头戏：资源管理器适配文件系统！

**Reply by User782Tec** [reply]
你咋不说文件系统适配资源管理器呢（bushi）
